tests/test_cargo_read_manifest.rs

```diff
@@ -1,22 +1,28 @@
+use regex::Regex;
 use support::{project, execs, main_file, basic_bin_manifest};
 use hamcrest::{assert_that};
 
 fn setup() {}
 
+fn remove_all_whitespace(s: &str) -> String {
+    let re = Regex::new(r"\s+").unwrap();
```

Comment:
Instead of using a regex for this, could this use `.split_whitespace().collect()`?